File: Web/Sigfrido/Ejemplos/php/json/MiObjeto.php
<?php
	class MiObjeto {
		var $nombre;
		var $origen;
		var $miCadena;
		var $array1=array();
		var $vector;
		public function __construct($nom, $ori, $cad){
			$this->nombre=$nom;
			$this->origen=$ori;
			$this->miCadena=$cad;
			array_push($this->array1, "adios");
			array_push($this->array1, 10);
			$this->vector=array();
			array_push($this->vector, "Elemento 1");
			array_push($this->vector, NULL);
			array_push($this->vector, "Elemento 3");
			array_push($this->vector, "Elemento 4");
		}
		
	}
	$cadena1 = new MiObjeto('Juan', 'Madrid', NULL);
	$myJSON = json_encode($cadena1);
	echo "<h3>";
	echo "JSON said: ".$myJSON."</h3>";
?>

File: Web/Sigfrido/Ejemplos/json/MiObjeto.java
import java.util.*;
import java.util.ArrayList;
import com.google.gson.Gson;

public class MiObjeto {
	private String nombre;
	private String origen;
	String miCadena;
	ArrayList<String> miColeccion = new ArrayList<String>();
	Vector miVector;
	private MiObjeto(String nombre, String origen, String cadena) {
		this.nombre = nombre;
		this.origen = origen;
		this.miCadena = cadena;
		miColeccion.add("adios");
		miColeccion.add("10"); //*
		miVector = new Vector();
		miVector.add("Elemento1");
		miVector.add(null);
		miVector.add("Elemento3");	
		miVector.add("Elemento4");
	}
	public static void main(String args[]) throws java.io.IOException {
		Gson gson = new Gson();
		MiObjeto obj = new MiObjeto("Juan", "Madrid", null);
		String jsonString = gson.toJson(obj);
		System.out.println("JSON: " + jsonString);
	}
}